From ea581eb70f6a57c3bfc600238a1c4ea0bf056872 Mon Sep 17 00:00:00 2001
From: Adrian Alonso <adrian.alonso@nxp.com>
Date: Mon, 18 Jan 2016 17:34:29 -0600
Subject: [PATCH 0930/5242] MLK-12307: ARM: imx: imx6q arm2 use lpddr2 in 2
 channel fixed mode

commit  f7acf0ec78362758e7a240c634d1dba44191bfbf from
https://source.codeaurora.org/external/imx/linux-imx.git

Set usable memory to 2 channel fixed mode
MMDC0 0x80000000 512M
MMDC1 0x10000000 512M
MMDC 2 channel fixed mode is enabled by setting
BOOT_CFG3[5:4] = 01 for Fixed 2x32 map

Signed-off-by: Adrian Alonso <adrian.alonso@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6q-pop-arm2.dts |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6q-pop-arm2.dts b/arch/arm/boot/dts/imx6q-pop-arm2.dts
index 1bd42f7..3be9076 100644
--- a/arch/arm/boot/dts/imx6q-pop-arm2.dts
+++ b/arch/arm/boot/dts/imx6q-pop-arm2.dts
@@ -17,7 +17,8 @@
 	compatible = "fsl,imx6q-pop-arm2", "fsl,imx6q";
 
 	memory {
-		reg = <0x10000000 0x40000000>;
+		linux,usable-memory = <0x10000000 0x20000000>,
+			<0x80000000 0x20000000>;
 	};
 
 	regulators {
-- 
1.7.9.5

